src/esp/metadata/managers/AssetAttributesManager.h

```diff
@@ -357,8 +357,8 @@ class AssetAttributesManager
   }  // AssetAttributeManager::getDefaultUVSphereTemplate
 
   /**
-   * @brief Return the spedified cube template.
-   * @param templateHndle The handle of the desired cube template. Verifies
+   * @brief Return the spedified UVSphere template.
```

Comment:
```suggestion
   * @brief Return the specified UVSphere template.
```
Same typo in other prim type comments.